.gitignore

```diff
@@ -0,0 +1,54 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
```

Comment:
Even though, you've added `*.pyc` here now, some files already added won't be removed from the repository.

To remove all pyc files and commit, do: `git rm --cached *.pyc; git add *.pyc; git commit -m "Remove pyc files."`